.github/workflows/WatchCondaForge.yml

```diff
@@ -37,7 +37,12 @@ jobs:
           title: "Update pangeo-notebook metapackage version to ${{ steps.latest_version.outputs.version }}"
           reviewers: "scottyhq"
           branch: "upgrade-pangeo-metapackage-version"
+          branch-suffix: timestamp
```

Comment:
I'm not sure we want this. Couldn't this lead to an accumulation of PRs (say over a weekend)?

Reply:
i don't think we'll be updating the metapackages that regularly. But worst case scenario we just close a couple of PRs. Or merge each separately so that there is a corresponding image for each metapackage version. Otherwise commits get force-pushed to the same PR and the image building wouldn't automatically retrigger because a new comment isn't made (see https://github.com/scottyhq/githubactions-test/pull/1)